tests/drivers/pwm/pwm_api/testcase.yaml

```diff
@@ -1,7 +1,5 @@
 tests:
   peripheral.pwm:
-    # FIXME: We should remove those and just rely on depends_on
-    platform_whitelist: quark_se_c1000_devboard quark_se_c1000_ss_devboard
-      arduino_101 arduino_101_sss colibri_imx7d_m4 sam_e70_xplained
     tags: drivers pwm
-    depends_on: pwm
+    filter: DT_ALIAS_PWM_0_LABEL or DT_ALIAS_PWM_1_LABEL or DT_ALIAS_PWM_2_LABEL or DT_ALIAS_PWM_3_LABEL
+    #depends_on: pwm
```

Comment:
Did you mean to comment this out?

Reply:
will be fixed, the yaml data is less complete than what we are getting with the filter, so was trying to get more coverage. adding pwm to all those remaining boards now...